src/content/snapshot/troubleshooting-snapshots.md

```diff
@@ -7,67 +7,61 @@ sidebar: { order: 3, label: "Troubleshooting" }
 
-- **Intentional randomness**: Some stories may render unpredictably intentionally. If this is the case you may want to [ignore the story](/docs/disable-snapshots#with-storybook) from UI Tests and move on.
-  If you still need inconsistent elements for local development purposes inside Storybook, you can use `isChromatic()` exported from [our package](/docs/ischromatic) to apply the solutions above only when in the Chromatic environment.
+Once you rerun a build, the subsequent build will feature a "Traces" column. This column links to the Trace Viewer for each snapshot in the build, with one link per enabled browser. Clicking on one of the browser buttons to open the Trace Viewer.
 
-### Serve static files
+![Build 2 was a rerun build so it has an additional "traces" column](../../images/view-trace.png)
 
-When using Playwright or Cypress, you can serve static files like fonts, images, and videos through your app server. This ensures that resources load consistently across all snapshots.
+### How to use the snapshot trace?
 
-For Storybook, use the [staticDirs](https://storybook.js.org/docs/configure/integration/images-and-assets#serving-stae-files-via-storybook-configuration) option to load static files for your stories.
+To generate these traces, Chromatic leverages Playwright's built-in [tracing capabilities](https://playwright.dev/docs/trace-viewer#network). Even if you're using Cypress or Storybook, Chromatic's uses Playwright to render and capture snapshots in its Capture Cloud. These traces capture network activity, console logs, DOM snapshots, and other debugging information.
 
-### Use fixed-height wrappers for portal components
+![Example of a Playwright trace capturing network requests and displaying the final DOM structure](../../images/trace-example.png)
 
-Portals allow components to render arbitrary elements outside the parent component's initial DOM hierarchy. For example, tooltips, modals, popovers, and menus can be triggered by a nested button, but render close to the top of the DOM hierarchy using portals.
+#### Network tab analysis
 
-With Playwright and Cypress, Chromatic snapshots the entire page, including modals, dialogs, and other portaled elements. However, with Storybook, portals render outside of Storybook’s DOM tree which can lead to cut-off snapshots.
+The network tab displays the resources loaded during capture, including fonts, stylesheets, scripts, and other assets. Check for resources that failed to load or took a long time. For example, if fonts aren't incorrect or styles are missing, ensure that the font or CSS files have loaded successfully with the correct MIME type. Consider loading slow assets [statically](#serve-static-files).
 
-<details>
-<summary>Why are snapshots of portal components (tooltip, modal, popover, menu) cut off?</summary>
+For a detailed list of trace viewer features, see the [Playwright documentation](https://playwright.dev/docs/trace-viewer#trace-viewer-features).
```

Comment:
The "For a detailed list of trace viewer features...." sentence feels like it should be something that could be inside an aside, wouldn't you agree?